CullFactory/Extenders/RadarTargetExtender.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using GameNetcodeStuff;
+using HarmonyLib;
+using UnityEngine;
+using static CullFactory.Plugin;
+
+namespace CullFactory.Extenders;
+
+public class RadarTargetExtender : MonoBehaviour
+{
+    public static int cachedradartarget { get; private set; }
```

Comment:
Since we're only counting one target, this probably wouldn't work very well with multiple monitors